app/internal/agenda_events.py

```diff
@@ -89,3 +90,22 @@ def get_events_in_time_frame(
     """Yields all user's events in a time frame."""
     events = get_all_user_events(db, user_id)
     yield from filter_dates(events, start_date, end_date)
+
+
+def make_dict_for_graph_data(db: Session, user_id: int) -> Dict[str, int]:
+    """create a dict with number of events per day for current week"""
+    WEEK_DAYS = 6
+    start_date, end_date = agenda.calc_dates_range_for_agenda(
+        0, 0, WEEK_DAYS)
+
+    events_this_week = get_events_per_dates(
+        db, user_id, start_date, end_date
+    )
+    events_for_graph = {
+        str(start_date + timedelta(i)): 0 for i in range(WEEK_DAYS + 1)
+    }
```

Comment:
This should be on a separate function

Reply:
this is the separate function.. this was the part i separated from the route.

Reply:
done